wp-content/themes/csisnuclearnetwork/template-parts/carousel.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * A template partial to output a carousel for the Nuclear Network default theme.
+ *
+ * @package CSIS iLab
+ * @subpackage @package NuclearNetwork
+ * @since 1.0.0
+ */
+
+
+
+?>
+
+<?php $featured_projects = get_field( 'featured_projects' ); ?>
+
+<?php if ( $featured_projects ) : ?>
+  <div id="splide" class="splide">
```

Comment:
Are we only using this template for the featured projects (I thought there a few other spots)? If so, you'll need to make this `id` field unique.

Another thing I might recommend doing is giving it a class of `js-splide` in addition to the `splide` class, and updating your JS file so it looks for elements with `js-splide` instead. That keeps your CSS tied to `splide` and JS to `js-splide`, so if for whatever reason you need to make changes to the names of one of those selectors, it doesn't break both functionalities.